client/post-editor/editor-sidebar/index.jsx

```diff
@@ -27,8 +29,31 @@ export default class EditorSidebar extends Component {
 		isPrivate: PropTypes.bool,
 	}
 
+	constructor() {
+		super();
+		this.state = { showFeedback: false };
+	}
+
+	openFeedbackPane = () => {
+		this.setState( { showFeedback: true } );
+	};
+	closeFeedbackPane = () => {
+		this.setState( { showFeedback: false } );
+	};
```

Comment:
Fun fact! We're only evaluating an expression so we can get rid of the noisy curlies!

```js
openFeedbackPane = () => this.setState( { showFeedback: true } )
```